main/src/ca/uwaterloo/flix/language/phase/Safety.scala

```diff
@@ -329,6 +337,32 @@ object Safety {
     case Predicate.Body.Loop(_, _, _) => Set.empty
   }
 
+  /**
+    * Variables that are bound in lattice position without `fix`.
+    */
+  private def forcedLatticeVariables(c0: TypedAst.Constraint): Set[Symbol.VarSym] =
+    c0.body.flatMap(forcedLatticeVariables).toSet
+
+  private def forcedLatticeVariables(p0: Predicate.Body): Set[Symbol.VarSym] = p0 match {
```

Comment:
Missing comment.

Reply:
Name suggestion for both `forcedLatticeVariables -> latticeVariablesOf`.

Reply:
maybe `noFixedLatticeVariablesOf`? `fix A(x; y)` doesn't add `y` to the result